src/Order.java

```diff
@@ -51,20 +51,8 @@ public void printOrderDetails() {
     }
 
     public double calculatePrice(String subscription) {
-        double totalPrice = 0.0;
+        priceCalculator = new PriceCalculator(subscription);
```

Comment:
Doesn't this need to be defined as PriceCalculator priceCalculator = new PriceCalculator(subscription)?